sanic/request.py

```diff
@@ -144,6 +145,9 @@ def __init__(
         self.transport = transport
         self.head = head
 
+        # enforce existing behavior prior to issue #2427
+        self._keep_blank_form_values = False
```

Comment:
Is there any specific reason to make this property private and then add a getter & setter without any logic in them?

Reply:
Missed that in refactoring.